src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+
+import java.util.Scanner;
+
+class Task {
+    String taskName;
+    boolean status;
```

Comment:
Perhaps you could name the status variable to make it sound like a boolean with is/can at the start.